azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/validator/AbstractConfigurationValidator.java

```diff
@@ -48,7 +48,7 @@ public String validateResourceGroup() {
     }
 
     public String validatePricingTier() {
-        if (mojo.getPricingTier() != null && AppServiceUtils.getPricingTierFromString(mojo.getPricingTier()) == null) {
```

Comment:
there are some more usages of getPricingTierFromString, shall we replace all of them?

Reply:
The method will be finally depredated after maven plugin track2 migration done